src/user/user.service.ts

```diff
@@ -33,6 +40,9 @@ export class UserService {
     }
 
     create({ password, ...userDTO }: User): Promise<User> {
+        if (!this.model.exists({ username: userDTO.username })) {
+            return null;
+        }
```

Comment:
This logic seems to be incorrect.
If user is not exist, then it should be able to be created.

Reply:
Actually no need to check this in backend, since `username` is set to be unique in database.

Reply:
ok